contrib_bots/lib/github.py

```diff
@@ -0,0 +1,24 @@
+# The purpose of this file is to handle all requests
+# to the Github API, which will make sure that all
+# requests are to the same account, and that all requests
+# authenticated correctly and securely
+# The sole purpose of this is to authenticate, and it will
+# return the requests session that is properly authenticated
+import requests
+import six.moves.configparser, os
+
+config_file = '~/.github-issue-bot/github-issue-bot.conf'
```

Comment:
By convention Python constants should be in capital letters and words should be separated by underscores:
https://www.python.org/dev/peps/pep-0008/#constants